lib/dap/filter/geoip2.rb

```diff
@@ -78,8 +88,9 @@ def decode(ip)
     unless @@geo_city
       raise "No MaxMind GeoIP2::City data found"
     end
-    return unless (geo_hash = @@geo_city.get(ip))
+
     ret = defaults
+    return unless (geo_hash = get_maxmind_data(@@geo_city, ip))
```

Comment:
```
geo_hash = get_maxmind_data(@@geo_city, ip)
return unless geo_hash 
```
Slightly easier to read here and below.  Defining the variable in the guard clause is kinda confusing.

Reply:
👍